web/src/main/client/src/Authenticator.js

```diff
@@ -0,0 +1,52 @@
+const base64 = require('base-64');
+
+let instance = null;
+
+class Authenticator {
+
+  constructor() {
+    if (!instance){
+      instance = this;
+    }
+
+    return instance;
+  }
+
+  authenticate(callback){
+    //Fetches the authentication token from the oauth api
+    fetch('https://happynews-api.svendubbeld.nl/oauth/token', {
+      method: 'post',
+      headers:{
+        "Authorization": "Basic " + base64.encode('happynews-editor:OuNNQtRGBIfUTG2IDICCdOUt'),
+        "Origin": window.location.hostname,
+        "Content-Type": "application/x-www-form-urlencoded; charset=utf-8"
+      },
+      body: 'grant_type=password&username=admin&password=password'
```

Comment:
Whoops i've misunderstood the flow of authentication 😳 ... Not gonna be able to fix this in the current sprint. I think it best if we put this in the next sprint and maybe give the login flow and account creation some more thought.